src/controllers/ProductsController.js

```diff
@@ -192,16 +192,17 @@ class ProductsController extends Controller {
 
   @httpVerb('get')
   @route('/v1/products/:productIDOrSlug/firmware')
-  async getFirmware(productIDOrSlug: string): Promise<*> {
+  async getFirmwares(productIDOrSlug: string): Promise<*> {
     const product = await this._productRepository.getByIDOrSlug(
       productIDOrSlug,
     );
     if (!product) {
       return this.bad('Product does not exist', 404);
     }
 
-    const firmwares = await this._productFirmwareRepository.getAllByProductID(
+    const firmwares = await this._productFirmwareRepository.getManyByProductID(
       product.product_id,
+      { take: 0 },
```

Comment:
This doesn't make sense. This should return zero items since `take` is zero

Reply:
hm, i thought it's standard convention in programming, if you pass 0 - it returns everything.
but ok. I have to remove default take value in the db.find() after that, this way it won't be confusing, does it make sense?